### PR TITLE
Small improvements to README file.

### PR DESCRIPTION
**Pull request by AndreeaEne**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,22 +1,21 @@
 # Home Assistant website
 
-This is the source for the [Home-Assistant.io website](https://home-assistant.io) for the [Home Assistant project](https://github.com/home-assistant/home-assistant)
+The source of the [Home-Assistant.io website](https://home-assistant.io), for the [Home Assistant project](https://github.com/home-assistant/home-assistant)
```
Comment (balloob) [inline README.markdown]: Let's stick with the start `This is the source for` but remove the `for the Home Assistant project` part.

Comment (AndreeaEne) [inline README.markdown]: Sure!

**Comment by robbiet480** [comment]
Thanks! 🐬 🍪 💯

**Comment by AndreeaEne** [comment]
You're very welcome 👍 !

**Pull request merged**